IOPool/Common/bin/EdmFileUtil.cpp

```diff
@@ -42,6 +42,7 @@ int main(int argc, char* argv[]) {
       "JSON,j", "JSON output format.  Any arguments listed below are ignored")("ls,l", "list file content")(
       "map,m", "Print TFile::Map(\"extended\"). The output can be HUGE.")("print,P", "Print all")(
       "verbose,v", "Verbose printout")("printBranchDetails,b", "Call Print()sc for all branches")(
+      "printClusters", "Print detailed information about baskets and clusters for all branches")(
```

Comment:
Does the `tree` option work with this? If so, maybe should extend the help message for `tree`?

Reply:
Yes, I intended to make it work with other trees than `Events` tree as well. Actually testing them now I see for Lumis and Runs it makes sense to include the metadata branches to the cluster size calculation, so I'll generalize this feature more.

I'll also extend the help message of `--tree`.

Reply:
Did the generalization. Now instead of skipping specific branches, the branches whose basket's boundaries do not align with cluster boundaries are detected, excluded from cluster size calculation from thereafter, and the branch names are printed at the end.